apps/chat/src/components/Chat/ChatMessage/ChatMessageContent/AssistantMessage.tsx

```diff
@@ -0,0 +1,114 @@
+import { memo } from 'react';
+
+import { useTranslation } from 'next-i18next';
+
+import classNames from 'classnames';
+
+import { isEntityNameOrPathInvalid } from '@/src/utils/app/common';
+
+import { Conversation } from '@/src/types/chat';
+import { Translation } from '@/src/types/translation';
+
+import { useAppSelector } from '@/src/store/hooks';
+import { SettingsSelectors } from '@/src/store/settings/settings.reducers';
+
+import { MessageAssistantButtons } from '@/src/components/Chat/ChatMessage/MessageButtons';
+import { AssistantSchema } from '@/src/components/Chat/ChatMessage/MessageSchema/AssistantSchema';
+import { MessageAttachments } from '@/src/components/Chat/MessageAttachments';
+import { MessageStages } from '@/src/components/Chat/MessageStages';
+import { ErrorMessage } from '@/src/components/Common/ErrorMessage';
+import ChatMDComponent from '@/src/components/Markdown/ChatMDComponent';
+
+import { LikeState, Message } from '@epam/ai-dial-shared';
+
+interface AssistantMessageProps {
+  message: Message;
+  conversation: Conversation;
+  isLastMessage: boolean;
+  isLikesEnabled: boolean;
+  withButtons?: boolean;
+  messageCopied?: boolean;
+  onCopy?: () => void;
+  onLike?: (likeStatus: LikeState) => void;
+  onRegenerate?: () => void;
+}
+
+const _AssistantMessage = ({
```

Comment:
I think it's better to use names without underscore something like `AssistantMessageView`